### ISSUE TITLE
modules not imported in Get-Office365Forwarders

### ISSUE THREAD

**Issue by adamsewell** [kieranwalsh/PowerShell#1]
I was testing your Get-Office365Forwarders script, and noticed that you check if the MSOnline and ExchangeOnlineManagement are installed but you never actually import them in your script causing the Connect-MsolService to fail as the function doesn't exist.

**Comment by kieranwalsh** [comment]
Oh, I've not had to import a module for years. Since PS 3, modules get imported automatically don't they? Which version of PS are you using?
It's possible you may have to close and re-open PowerShell before the updated modules become available though. Let me know if restarting the console works and I'll add a message in an updated version.

**Comment by adamsewell** [comment]
It must be something funky with the Windows Terminal, I tried it in the Powershell terminal and it worked fine.

**Comment by kieranwalsh** [comment]
Yes, I've not used that. Will try in the morning and then do a test on the host type - importing if needed. Thanks for letting me know.

**Comment by kieranwalsh** [comment]
I've updated it to work with Windows Terminal by importing the modules. Let me know if it works for you when you have a chance 👍.
